packages/core-data/src/selectors.ts

```diff
@@ -1129,7 +1131,10 @@ export function getAutosave< EntityRecord extends ET.EntityRecord< any > >(
 	}
 
 	const autosaves = state.autosaves[ postId ];
-	return find( autosaves, { author: authorId } ) as EntityRecord | undefined;
+
+	return autosaves?.find(
```

Comment:
Could potentially be `undefined` if we don't have any autosaves for the specified post.